### PR TITLE
Add confirmation prompt to server delete.

### PR DESCRIPTION
**Pull request by jcobol**
Add confirmation for server deletion.  See https://github.com/oracle/knife-bmcs/issues/17

Signed-off-by: john_l_hopkins john.l.hopkins@oracle.com

### CODE REVIEW

File: lib/chef/knife/bmcs_helper.rb [hunk 1]
```diff
@@ -93,6 +93,20 @@ def display_list(response, columns)
 
         warn_if_page_is_truncated(response)
       end
+
+      # Return a true or false with the confirmation result.
+      # Note: user prompt is bypassed with --yes to confirm automatically.
+      def confirm(prompt)
```
Comment (briangustafson) [inline lib/chef/knife/bmcs_helper.rb]: There's a ui.confirm operation, https://docs.chef.io/plugin_knife_custom.html - we should use that.

Comment (briangustafson) [inline lib/chef/knife/bmcs_helper.rb]: The code for this, just FYI, is at https://github.com/chef/chef/blob/master/lib/chef/knife/core/ui.rb#L268.

Comment (jcobol) [inline lib/chef/knife/bmcs_helper.rb]: I tried ui.confirm but to be honest, I wasn't happy with the implementation.

Comment (briangustafson) [inline lib/chef/knife/bmcs_helper.rb]: I think that either way is fine. If keeping the custom implementation, please make the y/n case insensitive.

**Pull request closed without merging**